frontend/js/i18n/de-lang.json

```diff
@@ -253,17 +252,17 @@
 		"tcp-forwarding": "TCP-Weiterleitung",
 		"title": "Streams",
 		"udp": "UDP",
-		"udp+pp": "UDP+proxy protocol",
-		"udp+tls": "UDP+TLS",
-		"udp+pp+tls": "UDP+proxy protocol+TLS",
+		"udp+pp": "UDP + proxy protocol",
+		"udp+tls": "UDP + TLS",
+		"udp+pp+tls": "UDP + proxy protocol + TLS",
```

Comment:
since there is not much space in the UI, I would prefer having no spaces in it (adding spaces would also need to be done in other langs) and for tcp